app/views/catalog/_index_marc.html.erb

```diff
@@ -68,88 +63,36 @@
-            Look for Online Versions
-          <% end %>
-        </div>
-      <% end %>
-    <% end %>
-
     <% if hathi_eta(document) && related_hathi_links(document).length > 0 %>
-      <div class="links results-item-section card">
-        <div class="card-header">
-          <div class="col-sm-7">
-            Online Access
-            <% if (Rails.application.config.x.suppress_browse != true) && (browse_sort_key = shelf_browse_sort_key_for(document)) %>
-              <span> | </span>
-              <span class="glyphicon glyphicon-book"></span> <%= link_to "Virtual Shelf Browse", shelfbrowse_path(:origin_sort_key => browse_sort_key) %>
-            <% end %>
-          </div>
-<!--          <div class="action-container col-sm-6">-->
-            <%#= link_to(JHConfig.params[:umlaut_base_url] +  "/resolve?#{document.export_as_openurl_ctx_kev}", {:class => "request btn btn-primary btn-sm"}  ) do %>
-<!--              Look for Other Versions-->
-            <%# end %>
-<!--          </div>-->
-        </div>
-
-        <div class="card-body">
-          <% if hathi_eta(document) && related_hathi_links(document).length > 0 %>
-            <% related_hathi_links(document).each do |url| %>
-              <%= link_to "Full text available from HathiTrust", url %><br/>
-            <% end %>
-          <% end %>
-        </div>
-
-      </div>
+      <%= render 'links_card', document: document %>
     <% end %>
-
+    
     <% if links_presenter.lines.present? && related_links_type(document) == :online %>
-      <div class="links results-item-section card">
-        <div class="card-header"><%= related_links_title(document) %>
-          <% if (Rails.application.config.x.suppress_browse != true) && (browse_sort_key = shelf_browse_sort_key_for(document)) %>
-            <span> | </span>
-            <span class="glyphicon glyphicon-book"></span> <%= link_to "Virtual Shelf Browse", shelfbrowse_path(:origin_sort_key => browse_sort_key) %>
-          <% end %>
-        </div>
-
-        <div class="card-body">
-          <% if hathi_eta(document) %>
-            <% related_hathi_links(document).each do |url| %>
-              <%= link_to "HathiTrust", url %><br/>
-            <% end %>
-          <% end %>
-          <%=
-          render :partial => "marc_display/presenter_content", :locals => {:presenter => links_presenter } if links_presenter.should_display?
-          %>
-        </div>
-
-      </div>
+      <%= render 'links_card', document: document %>
     <% end %>
-
+    
     <% if document.respond_to?(:to_holdings) && document.to_holdings.present? && related_hathi_links_etas_only(document).length == 0 %>
       <ul class="holdings list-group">
         <%= render :partial => "catalog/holding_expandable", :collection => document.to_holdings, :locals => {:document_id => document["id"], document: document  } %>
       </ul>
     <% end %>
+  <% if document.respond_to?(:export_as_openurl_ctx_kev) %>
```

Comment:
I think I added this section to add the COinS data, but replaced it with UnAPI

Reply:
@jonfackrell Should I just remove this section?